frontend/src/pages/nft/nft.tsx

```diff
@@ -13,9 +13,16 @@ import { getIpfsLink } from '@/utils';
 import { ListingCard } from './components';
 import { useNFT } from './hooks';
 import InfoSVG from './info.svg?react';
+import FeaturedPlayListSVG from './featured-play-list.svg?react';
+import MarkerSVG from './marker.svg?react';
 import styles from './nft.module.scss';
+import { useState } from 'react';
 
-const TABS = ['Overview', 'Properties', 'Activities'];
+const TABS: TabsProps['list'] = [
```

Comment:
Is there any practical advantage to specify `TabsProps['list']` type explicitly?

Since TS is smart enough to recognize it implicitly, for me it seems like it's cleaner to not specify it at all.

Reply:
It helps other developers quickly understand the expected data types for TABS and to have properties autocomplete here. This is especially useful when working in large projects with numerous data types.

But I agree that it is not necessary in this small case.